test/integration/connect/envoy/run-tests.sh

```diff
@@ -68,12 +68,25 @@ function init_workdir {
   return 0
 }
 
+# Delete a series of containers by their docker-compose name, without invoking
+# "docker-compose" which has a ~500ms per execution startup penalty.
+function docker_kill_rm {
```

Comment:
I suspect we'll save even more time by connecting directly to the docker API from Go instead of running the docker cli with exec. 

The current approach was a temporary step in the migration from bash to Go. We're exec'ing bash, then exec'ing other processes from bash. We could remove both of those by using the docker API directly.

Reply:
Oh most definitely. Mostly I was trying to shave a bit more time off just because we've accreted enough cases that they're taking like ~20min to run once which isn't awesome.